config/locales/en.yml

```
@@ -481,10 +481,10 @@ en:
     in_progress: in progress
     not_started: not started
   user_missing_organisation:
-    body: 'You don’t have an organisation assigned to your account. Contact a GOV.UK Forms super admin to assign an organisation to your account. You will then be able to use GOV.UK Forms.
+    body: 'Contact the GOV.UK Forms team to assign an organisation to your account.
 
       '
```

I don't think we need the newlines any more?

```suggestion
    body: Contact the GOV.UK Forms team to assign an organisation to your account.
```